packages/devtools_app/test/shared/eval_integration_test.dart

```diff
@@ -118,7 +118,12 @@ void main() {
                 (_) => throw Exception(
                   'The FutureFailedException was not thrown as expected.',
                 ),
-                onError: (Object? err) => err,
+                onError: (Object? err) {
+                  if (err is FutureFailedException) {
+                    return err;
+                  }
+                  throw err!;
+                },
```

Comment:
please remove these changes. You likely ran into a test flake if this failed on the bots.